app/views/posts/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1> Helllo post index is here</h1>
+
+<ul>
+  <% @posts.each do |post| %>
+    <li>
+    <%= post.title | %>
```

Comment:
When I try to run the project appears this error message:
- ActionView::SyntaxErrorInTemplate in PostsController#index